### PR TITLE
Table list html

### PR DESCRIPTION
**Pull request by muthusamyk**

### CODE REVIEW

File: web/src/modules/http/request.ts [hunk 1]
```diff
@@ -0,0 +1,65 @@
+import { HttpClient, HttpHeaders } from "@angular/common/http";
+import { Observable, throwError } from "rxjs";
+import { catchError, map } from "rxjs/operators";
+import * as _ from "lodash";
```
Comment (SandraSadan) [inline web/src/modules/http/request.ts]: Please import only relevant packages from lodash.

File: web/src/app/app.component.ts [hunk 2]
```diff
@@ -47,23 +51,41 @@ export class AppComponent {
...
-    this.data.forEach((value, index: number) => {
-      if (value['ID'] % 5 === 0) {
-        value['subtasks'] = this.data.splice(index+1, 4);
+  }
+
+  getColumn():void {
+    this.dataService.getColumn().subscribe({
+      next: (res) => {
+        this.columnList = get(res, 'column', []);
+        console.log(this.columnList);
```
Comment (SandraSadan) [inline web/src/app/app.component.ts]: Is this console log required?

Comment (muthusamyk) [reply web/src/app/app.component.ts]: no actually i will remove it

File: web/src/modules/http/request.ts (hunk 1 above)
Comment (muthusamyk) [reply web/src/modules/http/request.ts]: for this i will check and update my pr tomorrow

**Review by SandraSadan — APPROVED** [review]

**Pull request merged**